pepper-apis/src/main/resources/org/broadinstitute/ddp/db/dao/Statistics.sql.stg

```diff
@@ -0,0 +1,87 @@
+group Statistics;
+
+getParticipantStatistics() ::= <<
+select
+    case when ug.user_governance_id is null then 'self' else 'governed' end as user_type,
+    count(1) as user_count
+from
+    user_study_enrollment se
+    left join umbrella_study us on se.study_id = us.umbrella_study_id
+    left join user_governance ug on ug.participant_user_id = se.user_id
+    left join user_study_governance usg on usg.umbrella_study_id = us.umbrella_study_id
+        and usg.user_governance_id = ug.user_governance_id and ug.is_active = 1
+where
+    us.umbrella_id = :studyId and
+    se.enrollment_status_type_id = 2
```

Comment:
Should not hardcode type id here. Please do a subquery for the type id or do a `join enrollment_status_type` above.

Reply:
Joined